External/SDK/Windows/Windows10SDK.bff

```diff
@@ -1,10 +1,36 @@
 // Windows 10 SDK
 //------------------------------------------------------------------------------
-
-// Root Paths
-//------------------------------------------------------------------------------
-.Windows10_SDKBasePath          = '$_CURRENT_BFF_DIR_$/10'
-.Windows10_SDKVersion           = '10.0.17763.0'
+//
+// Detect Windows 10 SDK
+//
+// We search in the following locations, in order of preference:
+//  1) Vendorized in External (side by side with this bff)
+//  2) Part of a Windows 10 installation (Program Files)
+//
+#if file_exists( "Windows Kits\10\bin\10.0.17763.0\x64\RC.exe" )
```

Comment:
This isn't quite the correct directory ("Windows Kits" is not part of the existing structure), but I'll go ahead and fix that after merging